Documentation/cmdref/cilium-agent.md

```diff
@@ -198,6 +198,7 @@ cilium-agent [flags]
       --sidecar-istio-proxy-image string                     Regular expression matching compatible Istio sidecar istio-proxy container image names (default "cilium/istio_proxy")
       --single-cluster-route                                 Use a single cluster route instead of per node routes
       --skip-crd-creation                                    Skip Kubernetes Custom Resource Definitions creations
+      --skip-socket-lb-in-pod-ns                             Skip socket LB for services when inside a pod namespace, in favor of service LB at the pod interface. Socket LB is still used when in the host namespace.
```

Comment:
nit: could we stream-line this flag into e.g. `--bpf-lb-hostns-only=true` . Generic options we recently migrated over to have `bpf-lb-*` prefix.

Reply:
Done, renamed to `--bpf-socket-lb-hostns-only`.